anki-plugin/src/trmodels.py

```diff
@@ -336,8 +340,8 @@ class TiddlyRememberClozeTemplate(TemplateData):
             </div>
         """
 
-    name = "TiddlyRemember Cloze v1"
-    fields = ("Text", ID_FIELD_NAME, "Wiki", "Reference", "Permalink")
+    name = "TiddlyRemember Cloze v2"
```

Comment:
Like I mentioned, I don't want this to be a separate note type. I'll dive into implementing upgrades over the next couple of days and get back to you.